### PR TITLE
Add help for --template option

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Add help for --template option

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
ping @N3ll , @ikoevska , @vchimev

File: docs/man_pages/project/creation/create.md [hunk 1]
```diff
@@ -3,26 +3,32 @@ create
...
 Creates a new project for native development with NativeScript from the default template or from an existing NativeScript project.
 
 ### Options
 * `--path` - Specifies the directory where you want to create the project, if different from the current directory. The directory must be empty.
-* `--appid` - Sets the application identifier for your project. 
+* `--appid` - Sets the application identifier for your project.
 * `--copy-from` - Specifies a directory which contains an existing NativeScript project. If not set, the NativeScript CLI creates the project from the default hello-world template.
+* `--template` - Sets the source template for the project. The value can be anything that you can `npm install` - npm package, local path, .tgz, GitHub URL. The package will be used as `app` directory of the new application.
+<% if(isHtml) { %>
+If `--template typescript` or `--template tsc` is specified, the default TypeScript template (`tns-template-hello-world-tsc`) will be used.
```
Comment (vchimev) [inline docs/man_pages/project/creation/create.md]: :+1: just **tns-template-hello-world-ts**~~c~~

File: docs/man_pages/project/creation/create.md [hunk 2]
```diff
@@ -3,26 +3,32 @@ create
...
 Creates a new project for native development with NativeScript from the default template or from an existing NativeScript project.
 
 ### Options
 * `--path` - Specifies the directory where you want to create the project, if different from the current directory. The directory must be empty.
-* `--appid` - Sets the application identifier for your project. 
+* `--appid` - Sets the application identifier for your project.
 * `--copy-from` - Specifies a directory which contains an existing NativeScript project. If not set, the NativeScript CLI creates the project from the default hello-world template.
+* `--template` - Sets the source template for the project. The value can be anything that you can `npm install` - npm package, local path, .tgz, GitHub URL. The package will be used as `app` directory of the new application.
+<% if(isHtml) { %>
+If `--template typescript` or `--template tsc` is specified, the default TypeScript template (`tns-template-hello-world-ts`) will be used.
```
Comment (ligaz) [inline docs/man_pages/project/creation/create.md]: I think we should present this in the CLI help as well and not limit it to the HTML one.

**Comment by ikoevska** [comment]
ping @rosen-vladimirov

File: docs/man_pages/project/creation/create.md [hunk 3]
```diff
@@ -3,26 +3,31 @@ create
...
+For projects that target iOS, you can use uppercase or lowercase letters, numbers, and hyphens in the strings of the reversed domain name. Strings must be separated by a dot. For example: `com.nativescript.My-i0s-App`.
+* `<Template>` is a valid npm package which you want to use as template for your app. You can specify the package by name in the npm registry or by local path or GitHub URL to a directory or .tar.gz containing a package.json file. The contents of the package are be copied in the `app` directory of your project.<% } %>
```
Comment (rosen-vladimirov) [inline docs/man_pages/project/creation/create.md]: `are be copied` does not sound correct to me

**Comment by rosen-vladimirov** [comment]
:+1:  after fixing the comment :)

**Pull request merged**